modules/rosa-using-spot-instance.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-using-spot-instance_{context}"]
+= Adding a machine pool with spot instances to a cluster
+
+After a machine pool is created, you can configure additional machine pools with spot instances.
```

Comment:
Creating a machine pool first is not necessary.  Defining the use of spot instances is a part of the machine pool creation process.  You might be meaning to say "Additional machine pools can be created using spot instances".

Reply:
I guess `After a machine pool is created` should be `After a cluster is created`